eng/pipelines/runtimelab.yml

```diff
@@ -106,7 +106,7 @@ stages:
         isOfficialBuild: ${{ variables.isOfficialBuild }}
         timeoutInMinutes: 90
         testGroup: innerloop
-        buildArgs: -s nativeaot.objwriter+nativeaot+libs+nativeaot.packages -c release
+        buildArgs: -s nativeaot+libs+nativeaot.packages -c release
```

Comment:
These shortcuts should be reverted now that the PR is passing.

Reply:
The PR is passing, but it shouldn't. It has failing tests.